Do not generate *.man.txt files at build time. The build rule depends on the 
deprecated `col` command and the files are only needed for the MinGW platform.
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -3,9 +3,6 @@
 SUBDIRS = 
 include $(topdir)/autoconf/targets.mak
 
-all-targets: apcupsd.man.txt apcaccess.man.txt apctest.man.txt \
-             apccontrol.man.txt apcupsd.conf.man.txt
-
 apcupsd.man.txt: apcupsd.8
 	$(call MANIFY,$<,$@)
 apcaccess.man.txt: apcaccess.8
